lib/langParser.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const iso639 = require('iso-639-1')
+
+fs.readdir(`${__dirname}/../static/i18n`, function(err, items) {
+  var avaliableLanguages = {
+    Languages: {},
+  }
+  for (var i = 0; i < items.length; i++) {
+    var langName = items[i].substring(0, items[i].indexOf('.'))
+    var normalizedName = langName.split('-')[0]
+    var nativeName = iso639.getNativeName(normalizedName)
+    avaliableLanguages['Languages'][nativeName] = langName
+  }
+  fs.writeFileSync(
+    `${__dirname}/../src/js/languages.js`,
+    'export default ' +
+      JSON.stringify(avaliableLanguages) +
+      ' // eslint-disable-line'
```

Comment:
This is pretty sketch. Can we write directly to a `languages.json` JSON file instead?

Reply:
I've tried with no success...

Reply:
Is it possible to import a JSON file?

Reply:
https://blog.jonathanargentiero.com/how-to-load-a-json-content-inside-a-js-module-with-webpack/